releasenotes-dca/notes/add-k8s-gradual-rollout-support-315926f2d3c1b6d3.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Added support for gradual rollout in K8s SSI deployments.
```

Comment:
We likely need to highlight more to customers what they should expect to see from this change. Most notably, will this be enabled by default? Do they need remote config enabled or not? That sort of thing. Customers do read these, and they provide us something to point at if they get an unexpected result.